### DISCUSSION TITLE
Anki / SRS [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5545] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### No  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087078990147227718)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
